packages/labs/task/src/task.ts

```diff
@@ -86,13 +104,15 @@ export type StatusRenderer<R> = {
  */
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
 export class Task<T extends [...unknown[]] = any, R = any> {
-  private _previousDeps: T = [] as unknown as T;
+  private _previousArgs: T = [] as unknown as T;
   private _task: TaskFunction<T, R>;
-  private _getDependencies: DepsFunction<T>;
+  private _args: ArgsFunction<T>;
```

Comment:
It'd still prefix this with `_get` since it's a function to get the args, and not an array of the args like `_previousArgs`

Reply:
Done.